### DISCUSSION TITLE
Redirect url not correct

### DISCUSSION THREAD

**Discussion by freshface** [duncanmcclean/simple-commerce#1179] [Help] [answered]
This tag contains a redirect

` {{ sc:checkout x-data="checkoutForm" id="checkout-form" redirect="/checkout/complete" @submit.prevent="submit" }}`

But should it not contain the full url? Because it redirects to http://checkout/complete

My .env contains the `APP_URL=https://lively-pebble-mx3go0qkyw.ploi.dev/`

I am using mollie as payment provider (in test mode)

**Comment by duncanmcclean** [comment, marked as answer]
Does this work?

```
{{ sc:checkout x-data="checkoutForm" id="checkout-form" redirect="{config:app:url}/checkout/complete" @submit.prevent="submit" }}
```

**Reply by freshface** [reply]
Yes thx
